src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+body {
+  --main-color: #00acdc;
+
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  margin: 0;
+  box-sizing: border-box;
+}
+  
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)